public/doc/openapi-3-v1.0.0.json

```diff
@@ -1384,6 +1426,177 @@
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Icon not found"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Icon"
+        ],
+        "summary": "Edit an existing Icon",
+        "description": "Edits Icon specified by the given ID.",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/ID"
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Icon"
+              }
+            }
+          },
+          "description": "Parameters needed to update an Icon",
+          "required": true
+        },
+        "operationId": "updateIcon",
+        "responses": {
+          "200": {
+            "description": "Return the updated Icon object",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Icon"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "The Icon was not found"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Icon"
+        ],
+        "summary": "Delete an existing Icon",
+        "operationId": "destroyIcon",
+        "description": "Deletes the icon based on the icon ID passed\n",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/ID"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Icon deleted."
+          },
+          "404": {
+            "description": "Icon not found."
+          }
+        }
+      }
+    },
+    "/icons/{id}/icon_data": {
+      "get": {
+        "tags": [
+          "Icon"
+        ],
+        "summary": "Fetches the specified icon's image",
+        "operationId": "showIconData",
+        "description": "Fetch the specified portfolio item's icon image.",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/ID"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Portfolio Item Icon",
+            "content": {
+              "image/svg+xml": {
+                "example": "<svg rel=\"stylesheet\">thing</svg>"
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Icon Not Found/Not Present on Portfolio Item"
```

Comment:
Same with this one? Or does this one warrant more information since it's actually on the portfolio item? I feel like `"Icon not found."` is pretty self explanatory even if they're providing a portfolio item.